website/templates/project/project_header.mako

```diff
@@ -139,7 +139,18 @@
         % endif
 
         % if node['is_pending_embargo']:
-            <div class="alert alert-info">This ${node['node_type']} is currently pending registration, awaiting approval from project administrators. This registration will be final and enter the embargo period when all project administrators approve the registration or 48 hours pass, whichever comes first. The embargo will keep the registration private until the embargo period ends.</div>
+            <div
+                class="alert alert-info">This ${node['node_type']} is currently pending registration, awaiting approval from project administrators. This registration will be final and enter the embargo period when all project administrators approve the registration or 48 hours pass, whichever comes first. The embargo will keep the registration private until the embargo period ends.
+                % if 'admin' in user['permissions']:
+                        <div>
+                            <br>
+                            <button type="button" id="registrationCancelButton" class="btn btn-danger" data-toggle="modal" data-target="#registrationCancel">
+                                Cancel Registration
```

Comment:
As per osf-style, use sentence-case for button text. 

If this needs to be changed on any other "Cancel Registration" buttons, go ahead and change it in those places, too.